error from show tables after use hudi_catalog.db

<!-- (At least include the following, feel free to add if you have more content) -->

### Steps to reproduce the behavior (Required)
1. use hudi_catalog.db;
2. show tables:
````sql
mysql> show tables;
ERROR 1064 (HY000): com.google.common.util.concurrent.ExecutionError: java.lang.NoSuchMethodError: org.apache.orc.Reader.close()V
````

fe.warn.log:
```
2023-03-14 10:19:36,620 WARN (starrocks-mysql-nio-pool-1|215) [StmtExecutor.execute():552] execute Exception, sql show tables
com.google.common.util.concurrent.ExecutionError: com.google.common.util.concurrent.ExecutionError: java.lang.NoSuchMethodError: org.apache.orc.Reader.close()V
        at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache.get(LocalCache.java:3962) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3985) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4946) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$LocalLoadingCache.getUnchecked(LocalCache.java:4952) ~[spark-dpp-1.0.0.jar:?]
        at com.starrocks.connector.hive.CachingHiveMetastore.get(CachingHiveMetastore.java:388) ~[starrocks-fe.jar:?]
        at com.starrocks.connector.hive.CachingHiveMetastore.getTable(CachingHiveMetastore.java:203) ~[starrocks-fe.jar:?]
        at com.starrocks.connector.hive.HiveMetastoreOperations.getTable(HiveMetastoreOperations.java:58) ~[starrocks-fe.jar:?]
        at com.starrocks.connector.hudi.HudiMetadata.getTable(HudiMetadata.java:106) ~[starrocks-fe.jar:?]
        at com.starrocks.server.MetadataMgr.lambda$getTable$2(MetadataMgr.java:154) ~[starrocks-fe.jar:?]
        at java.util.Optional.map(Optional.java:215) ~[?:1.8.0_252]
        at com.starrocks.server.MetadataMgr.getTable(MetadataMgr.java:154) ~[starrocks-fe.jar:?]
        at com.starrocks.privilege.TablePEntryObject.generate(TablePEntryObject.java:111) ~[starrocks-fe.jar:?]
        at com.starrocks.privilege.DefaultAuthorizationProvider.generateObject(DefaultAuthorizationProvider.java:138) ~[starrocks-fe.jar:?]
        at com.starrocks.privilege.PrivilegeActions.checkAnyActionOnObject(PrivilegeActions.java:191) ~[starrocks-fe.jar:?]
        at com.starrocks.privilege.PrivilegeActions.checkAnyActionOnTable(PrivilegeActions.java:211) ~[starrocks-fe.jar:?]
        at com.starrocks.qe.ShowExecutor.lambda$handleShowTable$2(ShowExecutor.java:816) ~[starrocks-fe.jar:?]
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:174) ~[?:1.8.0_252]
        at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175) ~[?:1.8.0_252]
        at com.google.common.collect.CollectSpliterators$1WithCharacteristics.lambda$forEachRemaining$1(CollectSpliterators.java:67) ~[spark-dpp-1.0.0.jar:?]
        at java.util.stream.Streams$RangeIntSpliterator.forEachRemaining(Streams.java:110) ~[?:1.8.0_252]
        at com.google.common.collect.CollectSpliterators$1WithCharacteristics.forEachRemaining(CollectSpliterators.java:67) ~[spark-dpp-1.0.0.jar:?]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482) ~[?:1.8.0_252]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472) ~[?:1.8.0_252]
        at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708) ~[?:1.8.0_252]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_252]
        at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:566) ~[?:1.8.0_252]
        at com.starrocks.qe.ShowExecutor.handleShowTable(ShowExecutor.java:818) ~[starrocks-fe.jar:?]
        at com.starrocks.qe.ShowExecutor.execute(ShowExecutor.java:260) ~[starrocks-fe.jar:?]
        at com.starrocks.qe.StmtExecutor.handleShow(StmtExecutor.java:1102) ~[starrocks-fe.jar:?]
        at com.starrocks.qe.StmtExecutor.execute(StmtExecutor.java:510) ~[starrocks-fe.jar:?]
        at com.starrocks.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:348) ~[starrocks-fe.jar:?]
        at com.starrocks.qe.ConnectProcessor.dispatch(ConnectProcessor.java:462) ~[starrocks-fe.jar:?]
        at com.starrocks.qe.ConnectProcessor.processOnce(ConnectProcessor.java:728) ~[starrocks-fe.jar:?]
        at com.starrocks.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:69) ~[starrocks-fe.jar:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_252]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_252]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_252]
Caused by: com.google.common.util.concurrent.ExecutionError: java.lang.NoSuchMethodError: org.apache.orc.Reader.close()V
        at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2049) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache.get(LocalCache.java:3962) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3985) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4946) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$LocalLoadingCache.getUnchecked(LocalCache.java:4952) ~[spark-dpp-1.0.0.jar:?]
        at com.starrocks.connector.hive.CachingHiveMetastore.get(CachingHiveMetastore.java:388) ~[starrocks-fe.jar:?]
        at com.starrocks.connector.hive.CachingHiveMetastore.getTable(CachingHiveMetastore.java:203) ~[starrocks-fe.jar:?]
        at com.starrocks.connector.hive.CachingHiveMetastore.loadTable(CachingHiveMetastore.java:207) ~[starrocks-fe.jar:?]
        at com.google.common.cache.CacheLoader$FunctionToCacheLoader.load(CacheLoader.java:168) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.CacheLoader$1.load(CacheLoader.java:192) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3529) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2278) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2155) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2045) ~[spark-dpp-1.0.0.jar:?]
        ... 37 more
Caused by: java.lang.NoSuchMethodError: org.apache.orc.Reader.close()V
        at org.apache.hudi.common.util.OrcUtils.readAvroSchema(OrcUtils.java:249) ~[hudi-common-0.12.2.jar:0.12.2]
        at org.apache.hudi.io.storage.HoodieOrcReader.getSchema(HoodieOrcReader.java:81) ~[hudi-common-0.12.2.jar:0.12.2]
        at org.apache.hudi.common.table.TableSchemaResolver.readSchemaFromORCBaseFile(TableSchemaResolver.java:459) ~[hudi-common-0.12.2.jar:0.12.2]
        at org.apache.hudi.common.table.TableSchemaResolver.readSchemaFromBaseFile(TableSchemaResolver.java:645) ~[hudi-common-0.12.2.jar:0.12.2]
        at org.apache.hudi.common.table.TableSchemaResolver.fetchSchemaFromFiles(TableSchemaResolver.java:633) ~[hudi-common-0.12.2.jar:0.12.2]
        at org.apache.hudi.common.table.TableSchemaResolver.getTableParquetSchemaFromDataFile(TableSchemaResolver.java:266) ~[hudi-common-0.12.2.jar:0.12.2]
        at org.apache.hudi.common.table.TableSchemaResolver.getTableAvroSchemaFromDataFile(TableSchemaResolver.java:119) ~[hudi-common-0.12.2.jar:0.12.2]
        at org.apache.hudi.common.table.TableSchemaResolver.hasOperationField(TableSchemaResolver.java:564) ~[hudi-common-0.12.2.jar:0.12.2]
        at org.apache.hudi.util.Lazy.get(Lazy.java:53) ~[hudi-common-0.12.2.jar:0.12.2]
        at org.apache.hudi.common.table.TableSchemaResolver.getTableSchemaFromLatestCommitMetadata(TableSchemaResolver.java:223) ~[hudi-common-0.12.2.jar:0.12.2]
        at org.apache.hudi.common.table.TableSchemaResolver.getTableAvroSchemaInternal(TableSchemaResolver.java:191) ~[hudi-common-0.12.2.jar:0.12.2]
        at org.apache.hudi.common.table.TableSchemaResolver.getTableAvroSchema(TableSchemaResolver.java:140) ~[hudi-common-0.12.2.jar:0.12.2]
        at org.apache.hudi.common.table.TableSchemaResolver.getTableAvroSchema(TableSchemaResolver.java:129) ~[hudi-common-0.12.2.jar:0.12.2]
        at com.starrocks.connector.hive.HiveMetastoreApiConverter.toHudiTable(HiveMetastoreApiConverter.java:140) ~[starrocks-fe.jar:?]
        at com.starrocks.connector.hive.HiveMetastore.getTable(HiveMetastore.java:79) ~[starrocks-fe.jar:?]
        at com.starrocks.connector.hive.CachingHiveMetastore.loadTable(CachingHiveMetastore.java:207) ~[starrocks-fe.jar:?]
        at com.google.common.cache.CacheLoader$FunctionToCacheLoader.load(CacheLoader.java:168) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.CacheLoader$1.load(CacheLoader.java:192) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3529) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2278) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2155) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2045) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache.get(LocalCache.java:3962) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache.getOrLoad(LocalCache.java:3985) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$LocalLoadingCache.get(LocalCache.java:4946) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$LocalLoadingCache.getUnchecked(LocalCache.java:4952) ~[spark-dpp-1.0.0.jar:?]
        at com.starrocks.connector.hive.CachingHiveMetastore.get(CachingHiveMetastore.java:388) ~[starrocks-fe.jar:?]
        at com.starrocks.connector.hive.CachingHiveMetastore.getTable(CachingHiveMetastore.java:203) ~[starrocks-fe.jar:?]
        at com.starrocks.connector.hive.CachingHiveMetastore.loadTable(CachingHiveMetastore.java:207) ~[starrocks-fe.jar:?]
        at com.google.common.cache.CacheLoader$FunctionToCacheLoader.load(CacheLoader.java:168) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.CacheLoader$1.load(CacheLoader.java:192) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$LoadingValueReference.loadFuture(LocalCache.java:3529) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$Segment.loadSync(LocalCache.java:2278) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$Segment.lockedGetOrLoad(LocalCache.java:2155) ~[spark-dpp-1.0.0.jar:?]
        at com.google.common.cache.LocalCache$Segment.get(LocalCache.java:2045) ~[spark-dpp-1.0.0.jar:?]
        ... 37 more
```

### Expected behavior (Required)

### Real behavior (Required)

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`
```
mysql> select current_version();
+----------------------------+
| current_version()          |
+----------------------------+
| BRANCH-3.0-RELEASE 27ffa8b |
+----------------------------+
1 row in set (0.01 sec)
```

--- a/fe/fe-core/pom.xml
+++ b/fe/fe-core/pom.xml
@@ -492,6 +492,12 @@ under the License.
             <groupId>org.apache.spark</groupId>
             <artifactId>spark-sql_2.12</artifactId>
             <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <artifactId>orc-core</artifactId>
+                    <groupId>org.apache.orc</groupId>
+                </exclusion>
+            </exclusions>
         </dependency>
 
         <!-- https://mvnrepository.com/artifact/com.fasterxml.uuid/java-uuid-generator -->

